utils/utils.go

```diff
@@ -19,7 +20,15 @@ func OpenInBrowser(url string) error {
 	if err != nil {
 		return err
 	}
-	return run.PrepareCmd(browseCmd).Run()
+	err = run.PrepareCmd(browseCmd).Run()
+	if err != nil {
+		fmt.Printf("Error opening %s in your browser.\n", url)
```

Comment:
I don't think this error adds useful information. As part of error handling, the user will already see an error stating something like `exec: command "firefox" not found`. So I don't think we need to print this message at all